public/login/login.html

```diff
@@ -18,7 +18,7 @@
             <li id="notification"><a href="#"></a></li>
         </ul>
     </header>
-    <form class="login_form" id="loginForm" method="GET" action="/">
+    <form class="login_form" id="loginForm" method="POST" action="/">
```

Comment:
Avoid using form functionality, create functions in js that would handle the submit button. and make sure that you add "preventsDefault()" so that it won't do form's functionality.